File: src/main/kotlin/com/arnotjevleesch/skillquadrantback/repository/UserSkillsRepository.java
package com.arnotjevleesch.skillquadrantback.repository;

import com.arnotjevleesch.skillquadrantback.pojo.UserSkills;

import org.springframework.data.mongodb.repository.MongoRepository;


public interface UserSkillsRepository extends MongoRepository<UserSkills, String> {

    public UserSkills findByUser(String user);

}

File: src/main/kotlin/com/arnotjevleesch/skillquadrantback/repository/UserSkillsRepository.kt
package com.arnotjevleesch.skillquadrantback.repository

import com.arnotjevleesch.skillquadrantback.pojo.UserSkills

import org.springframework.data.mongodb.repository.MongoRepository


interface UserSkillsRepository : MongoRepository<UserSkills, String> {

    fun findByUser(user: String): UserSkills

}

